### PR TITLE
PAAS-6702 allow ignorin custom bad events to stop samson logic blocki…

### PR DESCRIPTION
**Pull request by grosser**
…ng deploys

some StatefulSets run into FailedCreate for example, it would be great if we don't have to rush out a global samson change to accommodate that while we still don't know if this is normal or caused by something else

@zendesk/compute

### CODE REVIEW

**Review by eatwithforks — APPROVED** [review]

File: plugins/kubernetes/README.md [hunk 1]
```diff
@@ -308,3 +308,11 @@ metadata.annotation.samson/force_update: "true"
 ### Static config per deploy group
 
 Set the kubernetes roles to `kubernetes/$deploy_group/server.yml` 
+
+### Ignoring warning events
+
+If a warning event stops deploys, but they are safe to ignore, add this:
+
+`metadata.annotations.samson/ignore_events="FailedCreate"`
```
Comment (uthark) [inline plugins/kubernetes/README.md:316]:
```suggestion
`metadata.annotations.samson/ignore_events="FailedCreate,AnotherEvent"` 
```

To show there might be multiple and show separator.

File: plugins/kubernetes/README.md [hunk 2]
```diff
@@ -308,3 +308,11 @@ metadata.annotation.samson/force_update: "true"
 ### Static config per deploy group
 
 Set the kubernetes roles to `kubernetes/$deploy_group/server.yml` 
+
+### Ignoring warning events
+
+If a warning event stops deploys, but they are safe to ignore, add this:
```
[the remark this answers is not in the document]
Comment (grosser) [reply plugins/kubernetes/README.md:314]: application owners

**Review by uthark — APPROVED** [review]

File: plugins/kubernetes/README.md (hunk 1 above)
Comment (grosser) [reply plugins/kubernetes/README.md:316]: good idea!

File: plugins/kubernetes/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (grosser) [reply plugins/kubernetes/README.md]: added to readme

**Pull request merged**